aidial_adapter_openai/gpt4_vision/chat_completion.py

```diff
@@ -270,15 +270,16 @@ async def chat_completion(
     api_version: str,
 ) -> Response:
     if request.get("n", 1) > 1:
-        raise HTTPException(
+        raise DialException(
             status_code=422,
             message="The deployment doesn't support n > 1",
             type="invalid_request_error",
+            display_message="This model doesn't support more than one alternative answer.",
```

Comment:
It's not something the chat user could fix _(yet, maybe we will add a feature to generate multiple responses later)_.